pom.xml

```diff
@@ -422,7 +423,7 @@
       <dependency>
         <groupId>org.apache.httpcomponents</groupId>
         <artifactId>httpclient</artifactId>
-        <version>4.3.1</version>
+        <version>${httpclient.version}</version>
```

Comment:
It seems like a mistake for us to declare both `org.apache.httpcomponents:httpclient` and `commons-httpclient:httpclient`. Are we adding versions for dependency management within our transitive dependencies, or are we actually using both ourselves?